### PR TITLE
feat(infra): gh action for deploying ceramic to production

### PR DESCRIPTION
**Pull request by nutrina**
This action should provision the ceramic node to production.

This closes: #481

### CODE REVIEW

File: .github/workflows/ceramic-promote-production.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        with:
+          cache: "yarn"
+          cache-dependency-path: infra/ceramic-mainnet/package-lock.json
+      # Update the pulumi stack with new image
+      - run: |
+          npm install
+          pulumi stack select -c gitcoin/ceramic/mainnet-node
+          pulumi config -s gitcoin/ceramic/mainnet-node set aws:region us-east-1 --non-interactive
+        working-directory: infra/ceramic-mainnet
+        env:
```
Comment (kammerdiener) [inline .github/workflows/ceramic-promote-production.yml:27]: I believe there are some additional values we need to pass in.

I know I include `ROUTE_53_ZONE` and `DOMAIN`

**Review by kammerdiener — COMMENTED** [review]
Just need to check that we are getting the correct ENVs through.

File: .github/workflows/ceramic-promote-production.yml (hunk 1 above)
Comment (nutrina) [reply .github/workflows/ceramic-promote-production.yml:27]: Those are passed in in the step below, as env vars

**Review by kammerdiener — APPROVED** [review]
I didn't read far enough down. LGTM!

**Pull request merged**